producer.go

```diff
@@ -98,8 +97,8 @@ func (p *Producer) consumeDeliveries() {
 				atomic.AddUint64(&stats.producerErr, 1)
 			}
 		} else {
-			p.log.Printf("Error consuming delivery event: Unknown event type (%s)", ev)
-			atomic.AddUint64(&stats.producerErr, 1)
+			p.log.Printf("Connection error (%s)", ev)
```

Comment:
Let's not assume that these are connection errors (there's no such guarantee anyway). The previous message looked fine to me.